recipes/smokeping/recipe.yaml

```diff
@@ -0,0 +1,140 @@
+# The app can be reached at http://<FQDN>/smokeping:8080
```

Comment:
Is this the correct URL to access it? To me i only get an UI error message because Angular thinks this is an app URL.

Reply:
It's working now.

Sadly the container is doing a redirect to `/smokeping` in the [httpd.conf](https://github.com/linuxserver/docker-smokeping/blob/c1c417e78d0d6c2277df216eb240654d8330525e/root/defaults/httpd.conf#L279), so that it is not possible to use the reverse proxy to support HTTPS.